### PR TITLE
Bump diagnostics for new fixes

### PR DESCRIPTION
**Pull request by SebastianMaj**
**Why**
Calls to Helium API were not going through because of missing header field
LOR should be a factor for PF

**How**
- Includes fix for API call to include headers
- Include LOR to be a factor for PF

**References**
[Helium API not returning due to missing headers](https://github.com/NebraLtd/hm-diag/pull/236)
[LOR should be a factor for PF](https://github.com/NebraLtd/hm-diag/pull/234)

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -58,7 +58,7 @@ services:
       - RELEASE_BUMPER=foobar
 
   diagnostics:
-    image: nebraltd/hm-diag:b1210ec
+    image: nebraltd/hm-diag:7e26f35
     environment:
       - FIRMWARE_VERSION=2021.11.17.2-1
```
Comment (marvinmarnold) [inline docker-compose.yml:63]: Also need to bump FIRMWARE_VERSION for hm-config and hm-diag.

**Review by marvinmarnold — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (SebastianMaj) [reply docker-compose.yml:63]: Should I update to FIRMWARE_VERSION=2021.11.17.2-2? or FIRMWARE_VERSION=2021.11.17.3-1?

Comment (SebastianMaj) [reply docker-compose.yml:63]: 2021.11.19.1? Sorry didn't realize it was the date at first 😂

Comment (marvinmarnold) [reply docker-compose.yml:63]: `2021.11.17.2-2`. The `2021.11.17.2` part is determined by the [most recent GA on hm-miner](https://github.com/NebraLtd/hm-miner/commit/b34655ff2c59a0ad00546d3339321cab336bc030). `-3` at the end is incremented each time we make a new release with the same GA.

Comment (marvinmarnold) [reply docker-compose.yml:63]: > 2021.11.19.1? Sorry didn't realize it was the date at first 

No worries. This is not well documented and we've been inconsistent about it in the past. That date is the date the last GA was released, which will not always correspond to additional releases we make on top of the GA.

Comment (SebastianMaj) [reply docker-compose.yml]: Ohhh ok that is more clear now, thank you! I bumped it up, should be good now

**Review by marvinmarnold — APPROVED** [review]

**Pull request merged**